MAINTAINERS.md

```diff
@@ -71,6 +71,8 @@ If you are an owner of the organization, you can see an automated list [here](ht
 - **cyqsimon ([@cyqsimon](https://github.com/cyqsimon))**: [28 December 2023](https://github.com/tldr-pages/tldr/issues/11864) — present
 - **Jongwon Youn ([@korECM](https://github.com/korECM))**: [29 December 2023](https://github.com/tldr-pages/tldr/issues/11892) — present
 - **Alejandro Cervera ([@tricantivu](https://github.com/tricantivu))**: [4 January 2024](https://github.com/tldr-pages/tldr/issues/11989) — present
+- **Mohammad Reza Soleimani ([@MrMw3](https://github.com/MrMw3))**:
+  [06 January 2024](https://github.com/tldr-pages/tldr/issues/12011) - present
```

Comment:
```suggestion
  [07 January 2024](https://github.com/tldr-pages/tldr/issues/12011) - present
```
Just a minor suggestion and this PR is GTG. The date mentioned here is the date on which the invitation was sent and not the date on which the issue was created.